### ISSUE TITLE
Latest versions of metasploit breaks rpc ssl/tls negotiation

### ISSUE THREAD

**Issue by ccammilleri** [SerpicoProject/Serpico#231]
Serpico generates generic msfprc connection errors when the user has supplied all the correct settings and msfprcd is running with SSL.

To reproduce bug:
1. install Kali 2016.2 and metasploit (v4.13.2-dev) and upgrade both to the latest version (as of 12/5/2016).
2. install serpico from deb package (release 1.1.0)
3. configure and run msfprcd with ssl
4. configure serpico for a metasploit connection
5. attempt to view hosts and witness error:
```
[!] MSF CONNECTION FAILED    

Connection reset by peer - SSL_connect
```

Appears somewhere between 4.12.40-dev and 4.13.2-dev SSLv3 support was dropped in msfrpcd and even though msfrpc-client is hard coded to TLS1, I still had to explicitly define option ssl_version to 'TLS1' to get it to negotiation.

## Fix

Explicitly define TLS option in server.rb
ex:
```
    opts = {
        :host => @msfoptions.ip,
        :port => @msfoptions.port,
        :user => @msfoptions.user,
        :pass => @msfoptions.pass,
        :ssl_version => 'TLS1'
    }
```

I'll self-assign and provide a fix soon.

**Comment by Petessss** [comment]
Test workaround. Works on 1.0 and 1.1.

**Comment by sobergods** [comment]
When starting Serpico using the start_serpico script, i get this error
serpico.rb:6:in `require': /opt/Serpico/Serpico/server.rb:81: syntax error, unexpected tSYMBEG, expecting '}' (SyntaxError)
	:ssl_version => 'TLS1'
	 ^
/opt/Serpico/Serpico/server.rb:82: syntax error, unexpected '}', expecting keyword_end
	from serpico.rb:6:in `<main>'

**Comment by ccammilleri** [comment]
@sobergods can you verify that you have commas in correct places within that code block? Please double check the above code snippet to make sure it looks exactly like yours.

**Comment by sobergods** [comment]
@ccammilleri @Petessss Thanks guys, it's working now, i added an extra ','
MSF RPC working now

**Comment by sobergods** [comment]
add Findings from Metasploit DB seems to be broken, i used Auto Add Vulnerabilities from Metasploit DB and it comes up empty, under METASPLOIT DATA MANAGEMENT, i have entries in Hosts and Vulnerabilities

**Comment by Petessss** [comment]
@sobergods 
 Do you have your vuln mappings setup? 
https://github.com/SerpicoProject/Serpico/wiki/Setting-up-Metasploit-RPC-Connection

**Comment by sobergods** [comment]
Thanks @Petessss i would need to map it manually, thanks guys

**Comment by Petessss** [comment]
Fixed in release 1.1.1.
